roles/pulp/tasks/install.yml

```diff
@@ -94,12 +94,10 @@
     - name: Reset ssh conn to allow user changes to affect when ssh user and pulp user are the same
       meta: reset_connection
 
-    - name: Create cache dir for Pulp
+    - name: Set permissions on {{ pulp_user_home }}
```

Comment:
https://pulp.plan.io/issues/5553

They are already, at least, on a fresh Pulp 3 install.  775 "vagrant" "apache" across the board.

The present issue is that if /var/lib/pulp/ is already present (because of an existing Pulp 2 install) then the whole directory is owned by "apache" with group "pulp" but due to the lack of group write permissions Pulp 3 can't write to /var/lib/pulp/

Reply:
With Pulp 2, what are the permissions?

It is a security practice to not relax permissions during software upgrades.

Unlike "others", the group might be an example where you can make an exception and relax permissions, if the group generally doesn't have arbitrary users on it.

Reply:
See Tanya's comment on the issue -- that's what the perms are